client/src/components/SeasonCollectionImageCardComponent/seasoncollectionimg.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import './seasoncollectionimg.css'
+import imageFile1 from "./Image 1.svg"
+import imageFile2 from "./Image 2.svg"
```

Comment:
Can you put these images in the assets directory and import them from there? (Import them in the `.stories.js` file, so you can pass them down as props)